packages/strapi-plugin-graphql/services/Query.js

```diff
@@ -230,7 +232,9 @@ module.exports = {
       ),
     );
 
-    return async (obj, options, { context }) => {
+    return async (obj, options = {}, { context }) => {
+      options = _.toPlainObject(options);
```

Comment:
It's a very bad idea to mutate the function params (It causes a lot of problems)
probably rename options to `_options` and create a `const options` variable